Frontend/src/app/views/dashboard/project-details/project-details.component.ts

```diff
@@ -254,6 +259,45 @@ export class ProjectDetailsComponent {
       this.toastrService.danger("There was an error updating the project", "Error");
       this.loadingProject = false;
     })
+    this.pageMode = 'view';
+  }
+
+  removeUserCheck(removeUserId: number) {
+    let title = "Are you sure you want to remove this user?";
+    let body = `Removing this user from the project will also remove all existing
+     time entries the user has created on this project.`;
```

Comment:
Maybe add the users name in here to make sure they are deleting the correct person?